katana/katana.py

```diff
@@ -90,6 +93,207 @@ def readconfig():
     cfg = json.loads("".join(lines))
     return cfg
 
+@route('/searchkw', method='POST')
+def searchkw():
+    """
+    This method returns the ActionFile Path of the selected Driver.
+    """
+    value = parseString("".join(request.body))
+    indented_xml = "".join(value.toprettyxml(newl='\n'))
+    corrected_xml = remove_extra_newlines_char_xml(indented_xml)
+    corrected_xml = corrected_xml
+    with open('obj.txt', 'w') as f:
```

Comment:
@Annam16 
I tried launching katana using
python katana/katana.py
When I tried to save wrapper keyword, it gives me
Traceback (most recent call last):
  File "/home/ka/Desktop/warriorframework/katana/bottle.py", line 861, in _handle
    return route.call(**args)
  File "/home/ka/Desktop/warriorframework/katana/bottle.py", line 1741, in wrapper
    rv = callback(*a, **ka)
  File "katana/katana.py", line 252, in parsexmlobj
    for line in io.open('kw_seq_template', 'r'):
IOError: [Errno 2] No such file or directory: 'kw_seq_template'
127.0.0.1 - - [22/Aug/2017 10:00:43] "POST /parsexmlobj HTTP/1.1" 500 853
{
  "response": "no"
}
127.0.0.1 - - [22/Aug/2017 10:00:45] "GET /checkfileexistwithsubdir/test.xml/testcase/none HTTP/1.1" 200 22